collector/interface.go

```diff
@@ -272,6 +296,10 @@ func processIfaceNetconfReply(reply *netconf.RPCReply, ch chan<- prometheus.Metr
 		newCounter(ch, ifaceDesc["OutputFifoErrors"], ifaceData.OutputErrorList.OutputFifoErrors.Text, ifaceLabels...)
 		for _, logIface := range ifaceData.LogicalInterfaces {
 			logIfaceLabels := []string{strings.TrimSpace(logIface.Name.Text)}
+			// set additional interface description metrics to an empty string for sub interfaces
```

Comment:
because we have to define the metric structure up front, we can't avoid adding the optional keys to logical interfaces, but we're not adding the values because we don't want to double-count physical/logical interfaces when summing/aggregating.